python/tests/test_liberate.py

```diff
@@ -5,4 +5,4 @@
 class TestLiberate(unittest.TestCase):
     def test_error_checking(self):
         with self.assertRaises(TypeError):
-            liberate.run_liberate(cell_names=[], char_tcl_path="/does/not/exist")
+            liberate.run_liberate(char_tcl_path="/does/not/exist")
```

Comment:
Since we added a public function to `liberate.py` should we either
1. make `run_liberate` private and tests the function that wraps it
2. duplicate tests for the function we added

Reply:
We don't really have tests for run_liberate, since it requires running liberate

Reply:
True, that would be nice to change in the future. There are ways to mock running liberate with certain exit codes etc.

Reply:
Ok so what are you proposing I change now?

Reply:
Nothing, can improve the test coverage for that file in the future if we want

Reply:
I will create an issue for it.